.github/workflows/build.yml

```diff
@@ -0,0 +1,39 @@
+name: FGD Build and Folder Copy
+on: 
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  build-windows:
+    runs-on: windows-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install srctools
+        run: .\install_srctools.bat
+      - name: FGD build and folder copy
+        run: .\build.bat all
+      - name: Artifact upload
+        uses: actions/upload-artifact@v2
+        with:
+          name: build_${{ runner.os }}-${{ github.sha }}
+          path: ./build/*.fgd
+          if-no-files-found: error
+  build-linux:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install srctools
+        run: chmod +x ./install_srctools.sh && bash ./install_srctools.sh
```

Comment:
We should probably commit this bash file permission. In Git Bash, do the chmod command, and you may also need to do `git update-index --chmod=+x install_srctools.sh`, and commit that.